packages/nodes-base/nodes/Contentful/resolveResponse.ts

```diff
@@ -0,0 +1,156 @@
+// @ts-nocheck
```

Comment:
This code is part of the official resolve package. Not sure how to integrate it. I didn't want to add a dependency for just one node.